_guides/2021-09-30-data-maturity-open-source.md

```diff
@@ -102,13 +102,13 @@ Data Maturity frameworks are useful frameworks for social sector organizations w
 To summarize, this guide sought to delineate common approaches to data maturity and examine common OSS applications and tools that many organizations can take at any point during their data journey. 
 
 ### Resources
+* [The FAIR Data Maturity Model](https://www.rd-alliance.org/system/files/FAIR%20Data%20Maturity%20Model_%20specification%20and%20guidelines_v1.00.pdf)
+* * CARE's [Responsible Data Maturity Model](https://careinternational.sharepoint.com/teams/Digital/Shared%20Documents/Forms/AllItems.aspx?id=%2Fteams%2FDigital%2FShared%20Documents%2FICT4D%2FArchive%2FResponsible%20Data%2FRD%20Maturity%20Model%2FDrafts%2FResponsible%20Data%20Maturity%20Model%2010%2E16%2E19%2Epdf&parent=%2Fteams%2FDigital%2FShared%20Documents%2FICT4D%2FArchive%2FResponsible%20Data%2FRD%20Maturity%20Model%2FDrafts&p=true&originalPath=aHR0cHM6Ly9jYXJlaW50ZXJuYXRpb25hbC5zaGFyZXBvaW50LmNvbS86YjovdC9EaWdpdGFsL0VlQVR5dUhNUVNGSW9pQnpnS0hWRkt3QnVSZ3dodlE4bUhnVGZsb0ZnbFMxV1E_cnRpbWU9bXA0ZzJjZkkyRWc)
```

Comment:
@bjk127 - any chance this deck is available somewhere that doesn't require a Microsoft sign in? And could you shorten the URL?